src/interface_layer/src/__main__.py

```diff
@@ -0,0 +1,269 @@
+# Import external tools
+from logging.handlers import RotatingFileHandler
+from logging import Formatter, DEBUG
+from flask import Flask, request
+from json import loads
+from configparser import ConfigParser as CP
+import logging
+from json import dumps
+from requests import post, Response
+# Import local tools
+from NgacApi.ngac import NGAC
+from result import Result, Ok, Error, unwrap, is_error
+from NgacApi import *
+from NgacApi.access_request import AccessRequest
+from NgacApi.user import User
+from NgacApi.resource import Resource
+from NgacApi.policy import Policy
+from NgacApi.parser import parse, get_user_attributes, get_objects_attributes, get_connection
+from result import to_error, Error
+from require import fields, response
+from NgacApi.attribute import ObjectAttribute
+
+# Import local files
+from admin.admin import *
+from admin import set_current_policy, get_policy
+
+
+# Set up logging
+logger = logging.getLogger(__name__)
+cfg = CP()
+# ------------------------------
+
+
+# Load the config file
+files_read = cfg.read("data/SERVER.ini")
+if len(files_read) == 0:
+    raise FileNotFoundError("Could not find data/SERVER.ini")
+# ------------------------------
+
+# Initiate the logger
+file_handler = RotatingFileHandler(
+    cfg["logging"]["folder"] + "/server.log",
+    maxBytes=int(cfg["logging"]["maxBytes"]),
+)
+file_handler.setLevel(cfg["logging"]["level"])
+file_handler.setFormatter(
+    Formatter("%(asctime)s %(levelname)s %(name)s %(threadName)s : %(message)s")
+)
+logger.addHandler(file_handler)
+logger.setLevel(DEBUG)  # cfg["logging"]["level"])
+# ------------------------------
+
+
+# Set up the flask app
+app = Flask(__name__)
+app.register_blueprint(admin)
+# ------------------------------
+
+
+# Set up the NGAC
+with open(cfg["NGAC"]["admin_token"], "r") as f:
+    admin_token = f.read()
+ngac = NGAC(token=admin_token, policy_server_url="http://130.240.200.92:8001")
+# ------------------------------
+
+
+# Setup ABE
+abe_url = cfg["ABE"]["url"]

```

Comment:
How about wrapping this in a dict or something? 
I think that would be preferable because it is a glob variable, and the fewer the better (to avoid name collisions etc.)
 If you still think it would be better to keep it as variables, it would be nice to have the names in CAPS to know that it is constants. (`abe_url`, `encrypt`, `decrypt`, `create_file`, `remove_file`)

Reply:
I kept them as globals because they allow me to use them at a "type" level, making the syntax for calls to the abe server cleaner.

using a dictionary would result in the following syntax
```python
response = ok(post(abe(abe_endpoints[`decrypt`]), data=fields))
```
instead of 
```python
response = ok(post(abe(decrypt), data=fields))
```
I prefer the latter. However, I do agree that the constants should be in capital letters.